browser/src/ContactPage.tsx

```diff
@@ -19,14 +19,14 @@ export default () => (
 
     <p>
       For questions about the OurDNA dataset, please see our blog.{' '}
-      <Link href="/news"></Link>.
+      <Link href="/news"/>.
```

Comment:
This is kind of useless Link, there is no text to present as underscore.
I guess you wanted this (link on the blog text):
`please see our <Link href="/news">blog</Link>.{' '}`